docs/examples/fork-process-model/flask-gunicorn/app.py

```diff
@@ -0,0 +1,44 @@
+import flask
+from flask import request
+
+from opentelemetry import trace
+from opentelemetry.instrumentation.flask import FlaskInstrumentor
+
+application = flask.Flask(__name__)
+
+FlaskInstrumentor().instrument_app(application)
+
+
+def fib_slow(n):
+    if n <= 1:
+        return n
+    return fib_slow(n - 1) + fib_fast(n - 2)
+
+
+def fib_fast(n):
+    nth_fib = [0] * (n + 2)
+    nth_fib[1] = 1
+    for i in range(2, n + 1):
+        nth_fib[i] = nth_fib[i - 1] + nth_fib[i - 2]
+    return nth_fib[n]
+
+
+@application.route("/fibonacci")
+def fibonacci():
+    tracer = trace.get_tracer(__name__)
```

Comment:
I could have put this in the global scope but doing that will lead no-op chain. Probably will update once something is decided about https://github.com/open-telemetry/opentelemetry-python/pull/1445